saltstack/base/salt/telemetry/init.sls

```diff
@@ -0,0 +1,18 @@
+{% set os = salt['environ.get']('OS') %}
+{% set cdp_telemetry_version = '0.1.0' %}
+{% set cdp_telemetry_rpm_repo_url = 'https://cloudera-service-delivery-cache.s3.amazonaws.com/telemetry/cdp-telemetry/'%}
+{% set cdp_telemetry_rpm_location = telemetry_rpm_repo_url + 'cdp_telemetry-' + cdp_telemetry_version + '.x86_64.rpm' %}
```

Comment:
Do you use this variable somewhere?

Reply:
my bad cdp_telemetry_rpm_location and cdp_telemetry_rpm_repo_url are mixed, also failed for me on testing